FreeType.xs

```diff
@@ -628,7 +628,7 @@ qefft2_face_fixed_sizes (Font_FreeType_Face face)
         int i;
         FT_Bitmap_Size *size;
         HV *hash;
-        double pt, ppem;
+        double pt = 12.0, ppem;
```

Comment:
Size = 0 would be the usual choice just if that is ever actually used, may crash(?) freetype. So any non-zero well to initialize, really.